recipes-app/iot2050-firmware-update/files/iot2050-firmware-update

```diff
@@ -5,55 +5,90 @@
 # Authors:
 #  Chao Zeng <chao.zeng@siemens.com>
 #  Jan Kiszka <jan.kiszka@siemens.com>
+#  Su Bao Cheng <baocheng.su@siemens.com>
 #
 # This file is subject to the terms and conditions of the MIT License.  See
 # COPYING.MIT file in the top-level directory.
 #
+# To use this tool, an update package in <firmware-update-package>.tar.xz format are needed.
+#
+# The <firmware-update-package>.tar.xz contains below files:
+#   - firmware.bin: The firmware to update, could be more than one.
+#   - update.conf.json: The update criteria.
+#
+# Example of update.conf.json:
+# {
+# 	"firmware": [
+# 		{
+# 			"description": "[optional] bla bla bla",
+# 			"name": "pg1-basic-vx.y.z.bin",
+# 			"version": "[optional] whatever",
+# 			"target_board": "pg1-basic"
+# 		},
+# 		{
+# 			"description": "[optional] bla bla bla",
+# 			"name": "pg2-advanced-vx.y.z.bin",
+# 			"version": "[optional] whatever",
+# 			"target_board": "pg2-advanced",
```

Comment:
target_boards - making the value an array. Will save us from copy&pasting the entry for every board group.

Reply:
OK

Reply:
Done, please review again.